ipam/allocator.go

```diff
@@ -113,7 +113,20 @@ func NewAllocator(config Config) *Allocator {
 
 // Start runs the allocator goroutine
 func (alloc *Allocator) Start() {
-	alloc.loadPersistedData()
+	if alloc.loadPersistedData() {
+		if len(alloc.seed) != 0 {
+			alloc.infof("Found persisted IPAM data, ignoring supplied IPAM seed")
+		}
+	} else {
+		if len(alloc.seed) != 0 {
+			alloc.infof("Initialising with supplied IPAM seed")
+			alloc.createRing(alloc.seed)
+		} else if alloc.paxos.IsElector() {
+			alloc.infof("Initialising via deferred consensus")
+		} else {
+			alloc.infof("Initialising as observer - awaiting IPAM data from another peer")
+		}
```

Comment:
I know this is from a refactor... but this inner `if` really should be a `switch`.